README.md

```diff
@@ -0,0 +1,10 @@
+# l2switch
+This branch is a mirror of the OpenDaylight l2switch gerrit project
```

Comment:
Can we add a comment that it is setup to use the Aluminum release of OpenDayLight. And a note that this removes a dependency on leveldbjni, allowing for the controller to run on ARM32 architectures (e.g., Raspberry Pi).